docker/recorder/src/test/java/org/arquillian/cube/docker/impl/client/utils/NumberConversionTest.java

```diff
@@ -0,0 +1,36 @@
+package org.arquillian.cube.docker.impl.client.utils;
+
+import org.junit.Test;
+
+import static org.arquillian.cube.docker.impl.client.utils.NumberConversion.convertToLong;
+import static org.arquillian.cube.docker.impl.client.utils.NumberConversion.humanReadableByteCount;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+public class NumberConversionTest {
+
+    @Test
+    public void convert_to_long_test() {
+        assertThat(convertToLong((byte)127), is(127L));
+        assertThat(convertToLong(null),is(0L));
+
+        assertThat(convertToLong((short) 32767), is(32767L));
+        assertThat(convertToLong(100L),is(100L));
+
+        assertThat(convertToLong(2147483647), is(2147483647L));
+        assertThat(convertToLong(0x10), is(16L));
+    }
+
+    @Test
+    public void human_readable_byte_count(){
```

Comment:
Last thing before merging. Can you split each assert in its own test? For example in this test you are checking that different conversions are possible. KB, MB, TB, .... so I would write tests like:

```
should_be_possible_to_show_megabytes_in_human_readable_form()
should_be_possible_to_show_gigabytes_in_human_readable_form()
...
```